packages/node/src/types/config.ts

```diff
@@ -148,6 +148,58 @@ export type LocalEvaluationConfig = {
    * evaluation.
    */
   assignmentConfig?: AssignmentConfig;
+
+  /**
+   * To use streaming API or polling. With streaming, flag config updates are
+   * received immediately, no polling is necessary. If stream fails for any
+   * reason, it will fallback to polling automatically.
+   * Default will be false, using poller.
+   */
+  streamUpdates?: boolean;
+
+  /**
+   * The stream server endpoint from which to stream data.
+   */
+  streamServerUrl?: string;
+
+  /**
+   * To use with streaming. The timeout for connecting an server-side event
+   * stream. Aka, the timeout for http connection.
+   */
+  streamConnTimeoutMillis?: number;
+
+  /**
+   * To use with streaming. The timeout for a single attempt of establishing
+   * a valid stream of flag configs.
+   * This includes streamConnTimeoutMillis and time for receiving initial
+   * flag configs.
+   */
+  streamFlagConnTimeoutMillis?: number;
+
+  /**
+   * To use with streaming. The number attempts to connect before declaring
+   * streaming fatal error.
+   */
+  streamFlagTryAttempts?: number;
+
+  /**
+   * To use with streaming. The delay between attempts to connect.
+   */
+  streamFlagTryDelayMillis?: number;
+
+  /**
+   * To use with streaming. The base delay to retry streaming after stream
+   * fatal error and fallbacked to poller.
+   */
+  streamFlagRetryDelayMillis?: number;
+
+  /**
+   * To use with streaming. The jitter to add to the delay for retry streaming
+   * after stream fatal error and fallbacked to poller.
+   * A random number between 0 and streamFlagRetryJitterMillis will be added to
+   * streamFlagRetryDelayMillis as the delay.
+   */
+  streamFlagRetryJitterMillis?: number;
```

Comment:
Added this jitter config.

Reply:
I think we can combine `streamConnTimeoutMillis` and `streamFlagConnTimeoutMillis` into one. Remove `streamFlagRetryDelayMillis` to be just default 15 seconds. Remove `streamFlagRetryJitterMillis` to just be 2 seconds jitter.

Reply:
100% agreed. Also I think the difference between try and retry is a little unlclear, I think we only expose one of those config types. most likely just the try configs and hardcode the retry configs ourselves. 

So:
1. streamConnTimeoutMillis
2. streamFlagRetryDelayMillis